### DISCUSSION TITLE
Welcome to Group Income Discussions!

### DISCUSSION THREAD

**Discussion by taoeffect** [okTurtles/group-income#1057] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dcm03** [comment]
My name is Deep and I am from India. I have been amazed with basic income which will resolve an issue for working age. I will make the flowing generation to become free to get off from the mindset of working for income. There is is a huge abundance out in the world and to become one of the best for humanity it is necessary for every living being should have and opportunity to  explore beyond their abilities and help really contributing for their true talent to make choices. Group income will help as a strong bridge from beginning till the end.

**Reply by taoeffect** [reply]
Hi @dcm03, welcome and thanks for your comment! :)

We're busy bringing the finishing touches on the software, and if you'd like to try it out feel free to give it a spin based on what's currently on Github and let us know your thoughts!

**Comment by Newman5** [comment]
Group income rules!  Hey Taoeffect, the web brings me here and I must comment to simply say, "Hi!" and to share kudos for nurturing this idea and project.  Well done!  I'm subscribed on YouTube and you have my email address.  I'll keep an eye out for updates!

**Reply by taoeffect** [reply]
Thanks for the kind words @Newman5! 😄
